### PR TITLE
rajkumar42/enableattachtemplate

### PR DESCRIPTION
**Pull request by rajkumar42**
Enabling process picker for attach.

### CODE REVIEW

**Comment by rajkumar42** [comment]
@DustinCampbell @caslan @chuckries @wesrupert

File: package.json [hunk 1]
```diff
@@ -502,9 +502,18 @@
                 "default": "The process name to attach to. If this is used, 'processId' should not be used."
               },
               "processId": {
-                "type": "integer",
-                "description": "The process id to attach to. If this is used, 'processName' should not be used.",
-                "default": 0
+                "anyOf": [ 
+                    {
+                      "type": "string",
+                      "description": "Lists running processes to attach to. If this is used, 'processName' should not be used.",
```
Comment (caslan) [inline package.json]: How about: "The process id to attach to. Use "${command.pickProcess}" to get a list of running processes to attach to. If this is used, 'processName' should not be used.

**Comment by caslan** [comment]
Otherwise, LGTM

**Comment by wesrupert** [comment]
:+1:

**Pull request merged**